set-version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ex
+
+# This file is responsible for updating the project (and Keycloak image) version in the release pipeline.
+# It has been copy-pasted from https://github.com/keycloak/keycloak-gatekeeper/blob/master/set-version.sh
+
+NEW_VERSION=$1
+
+CURRENT_VERSION=`awk '/Version.*=/ { print $3 }' version/version.go | sed 's/"//g'`
+sed -i "s/$CURRENT_VERSION/$NEW_VERSION/g" version/version.go
```

Comment:
Make sed commands more specific, replacing the exact line (ignoring current version set)

Reply:
@slaskawi +1. Would be better to know the exact pattern, the `sed` is replacing.

Also since the following three lines are identical, would it be possible to change it into a for loop?

Something like the (untested) snippet below:
```
declare -a UPDATE_VERSION_FILE_LIST=("version/version.go", ..., "deploy/operator.yaml")
for file in "${UPDATE_VERSION_FILE_LIST[@]}"
do
  sed -i "s/$CURRENT_VERSION/$NEW_VERSION/g" "$file"
done
```

Reply:
I came up with a function. Now it should be better.